config/StrategyResolver.py

```diff
@@ -156,69 +156,47 @@ def confirm_harvest(self, before, after, tx):
             "bveCVX", "badgerTree"
```

Comment:
There's an assert above these lines that might need to be updated

https://github.com/sajanrajdev/strategy-convex-stables/blob/9dd7af59526feb6808fa00d78a85ebd3199ab8a2/config/StrategyResolver.py#L140-L149

Reply:
Cool, got it. This looks fine then. 
Just these magic numbers can sometime get a bit confusing. I'm guessing what happened is that earlier 20% was autocompouded and the rest 80% (performance fee + badger tree) was deposited into the vault. Whereas now, even though there's no autocompounding, the 20% performance fee is taken directly in the token and rest 80% is deposited into the vault for badger tree. So even though we made some changes to the strat, the test still works because in both cases 80% is deposited into the vault

Reply:
That's correct, which is what the test intended to check. I just pushed an small improvement and added the CRV portion received by the strat to make it more accurate (I considered cvxCRV 1:1 with CRV for simplicity).